### PR TITLE
Joslobo/add microsoft repo

### PR DESCRIPTION
**Pull request by jslobodzian**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?

###### Change Log  

- Change
- Change
- Change

###### Does this affect the toolchain?  

**YES/NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Pipeline build id: xxxx

### CODE REVIEW

File: toolkit/resources/manifests/package/toolchain_aarch64.txt [hunk 1]
```diff
@@ -267,12 +267,12 @@ make-4.2.1-5.cm1.aarch64.rpm
...
-mariner-repos-extras-preview-1.0-13.cm1.noarch.rpm
-mariner-repos-preview-1.0-13.cm1.noarch.rpm
-mariner-repos-ui-1.0-13.cm1.noarch.rpm
-mariner-repos-ui-preview-1.0-13.cm1.noarch.rpm
+mariner-repos-1.0-14.cm1.noarch.rpm
+mariner-repos-extras-1.0-14.cm1.noarch.rpm
+mariner-repos-extras-preview-1.0-14.cm1.noarch.rpm
+mariner-repos-preview-1.0-14.cm1.noarch.rpm
+mariner-repos-ui-1.0-14.cm1.noarch.rpm
+mariner-repos-ui-preview-1.0-14.cm1.noarch.rpm
```
Comment (PawelWMS) [inline toolkit/resources/manifests/package/toolchain_aarch64.txt:275]: I think you need to add the two new subpackages here - we list all packages built in the toolchain in this file, if I'm not mistaken.

Comment (PawelWMS) [reply toolkit/resources/manifests/package/toolchain_aarch64.txt:275]: You'd also need to then also update `toolkit/resources/manifests/package/update_manifests.sh`.

Comment (jslobodzian) [reply toolkit/resources/manifests/package/toolchain_aarch64.txt:275]: Interesting...why would we do that?  Seems like that causes a circular dependency between the toolchain and the downstream repos.

Comment (jslobodzian) [reply toolkit/resources/manifests/package/toolchain_aarch64.txt:275]: Updated.

**Review by anphel31 — APPROVED** [review]

File: toolkit/resources/manifests/package/toolchain_aarch64.txt (hunk 1 above)
Comment (PawelWMS) [reply toolkit/resources/manifests/package/toolchain_aarch64.txt:275]: Thank you, approved. I didn't get the comment about circular dependencies, though. If you have time and think there's a bug here we should address, let's talk offline.

**Review by PawelWMS — APPROVED** [review]

**Pull request merged**